src/Raven.Client/Documents/Operations/AI/GenAiConfiguration.cs

```diff
@@ -31,6 +31,8 @@ public class GenAiConfiguration : AbstractAiIntegrationConfiguration
 
     public int MaxConcurrency { get; set; } = DefaultMaxConcurrency;
 
+    public bool ProcessNewDocumentsOnly { get; set; } = true;
```

Comment:
I really don't like this here. 
Naming wise: What is _new document_ here ? 

Also - that isn't a configuration for the task - that is basically what is the current etag that is starts from, and that should be a parameter for the creation, not persistent in the configuration like this.